homeassistant/components/matter/adapter.py

```diff
@@ -96,8 +97,14 @@ def _create_device_registry(
         device_type_instances = node_device.device_type_instances()
 
         name = basic_info.nodeLabel
-        if not name and isinstance(node_device, MatterBridgedNodeDevice):
-            # fallback name for Bridge
+        # Add a better way to detect if a device is a bridge,
+        # for now we just check if the node has the aggregator_device_type_instance variable
+        if (
+            not name
+            and isinstance(node_device, MatterNodeDevice)
```

Comment:
This check is not needed. `node()` is defined on `AbstractMatterNodeDevice`: https://github.com/home-assistant-libs/python-matter-server/blob/main/matter_server/common/models/node_device.py#L21-L22